Userland/Libraries/LibWeb/HTML/HTMLElement.idl

```diff
@@ -4,6 +4,7 @@
 #import <HTML/HTMLOrSVGElement.idl>
 #import <DOM/Element.idl>
 #import <DOM/EventHandler.idl>
+#import <HTML/HTMLOrSVGElement.idl>
```

Comment:
This import is defined twice, albeit not in alphabetical order... you could fix that instead!

Reply:
True, this doesn't need to be defined twice